python/ray/ml/utils/datasets.py

```diff
@@ -0,0 +1,59 @@
+from typing import Optional, Tuple, Union
+from ray.data import Dataset
+
+
+def train_test_split(
+    dataset: Dataset,
+    test_size: Union[int, float],
```

Comment:
I noticed that this deviates from the sklearn `train_test_split` [API](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html) in that there's no `train_size` arg, any particular reason for that? My guess is that having both is redundant unless you want to explicitly support subsampling (e.g. `train_test_split(test_size = 0.1, train_size=0.4)`), since `train_size` can always be the complement.

Reply:
I decided to remove it for simplicity, as the validation code can get very extensive with both arguments. We can add it back in, but that would make the function a lot more complicated.

Reply:
Yeah understood, that sounds good!